packages/block-library/src/group/edit.js

```diff
@@ -52,7 +57,14 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 	const { type = 'default' } = usedLayout;
 	const layoutSupportEnabled = themeSupportsLayout || type !== 'default';
 
-	const blockProps = useBlockProps();
+	const classes = classnames( className, {
+		'is-layout-flex': type === 'flex',
```

Comment:
Should this be a `is-layout-{ type }` for consistency?

Reply:
I guess it would accomplish the same, right? How would I write the left side of that statement, though? 🙈

Reply:
Nice one. Pushed!